templates/zerver/help/make-an-announcement.md

```diff
@@ -0,0 +1,34 @@
+# Making an Announcement
+
+An announcement on Zulip is a message which will reach
+ all users organization-wide.
+Announcements on Zulip are sent using the `#announce` stream.
+
+1. To send an announcement, click the **New stream message**
+ button located at the bottom of your screen.
+ A box similar to the one shown in the below image will appear.
+
+    ![New stream message](/static/images/help/new-stream.png)
+
+3. In the **Stream** field, input `announce` to select the `#announce` stream.
+
+4. Enter the topic for your announcement in the **Topic** field.
+ Your topic name can be no longer than 52 characters.
+ 
+5. Finally, enter your announcement in the box labeled
+ **Compose your message here...** or messaging box.
+ 
+6. You can now send your announcement under the specified topic.
+
+You can always delete your announcement by
+ clicking the **x** (![x](/static/images/help/x.png)) icon located at
+ the top-right corner of your messaging box.
+ 
+Your announcement will look similar to the below image.
+
+![Example announcement](/static/images/help/announcement.png)
+
+By default all users are subscribed to the `#announce` stream but users can
```

Comment:
This can be formatted as 
```
!!! warn ""
    Note: By default, all users are subscribed to the `#announce` stream, but users can unsubscribe from and mute the stream. This means that announcements, although sent organization-wide, will not reach users who have muted or unsubscribed from the `#announce` stream.
```
Make sure to do a line wrap and that the line wrapped characters have the same indentations.